demo-app/src/open-truss/components/IncrementAccountID.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  BaseOpenTrussComponentV1PropsShape,
+  computed,
+} from '@open-truss/open-truss'
+import { AccountIDsSignal } from '../signals'
+import { type z } from 'zod'
+import { useState } from 'react'
+
+export const Props = BaseOpenTrussComponentV1PropsShape.extend({
+  accountIds: AccountIDsSignal,
+})
+
+export default function IncrementAccountID({
```

Comment:
small nit: we're not incrementing anything here so maybe a better name is `CollectAccountID`?

Reply:
lol you're totally right. I think in my mind it started with just incrementing, but I wanted to use ya'll's ids in the demo, so I changed it.